source/features/quick-repo-deletion.css

```diff
@@ -9,3 +9,68 @@
 	opacity: 30%;
 	z-index: 91; /* Higher than sticky readme header and `sticky-sidebar` #4192 */
 }
+
+.rgh-quick-repo-deletion:not([open]) .btn::after {
+	content: 'Delete';
+}
+
+.rgh-quick-repo-deletion[open] .btn {
+	animation: 6s step-end forwards rgh-quick-repo-deletion-scale;
+}
+
+.rgh-quick-repo-deletion[open] .btn::after {
+	animation: 6s step-end forwards rgh-quick-repo-deletion-countdown;
+	content: '';
+}
+
+@keyframes rgh-quick-repo-deletion-scale {
```

Comment:
Not an upgrade. More code and less guarantee that the CSS will sync up (hint: it won't). This CSS file must be restored in its entirety

Reply:
Just an experiment 😅 was planning to revert to using JS